IPython/core/completer.py

```diff
@@ -1371,15 +1371,14 @@ def _jedi_matches(self, cursor_column:int, cursor_line:int, text:str):
                 else:
                     raise ValueError("Don't understand self.omit__names == {}".format(self.omit__names))
 
-        interpreter = jedi.Interpreter(
-            text[:offset], namespaces, column=cursor_column, line=cursor_line + 1)
+        script = jedi.Interpreter(text[:offset],namespaces)
```

Comment:
Why did you change the name here from `interpreter` to `script`?

Reply:
Sorry, it’s my mistake.
I will fix it.

Reply:
When I changed the variable name to “interpreter”, some checks were not successful.
Why?

Reply:
Thanks for the change! 

About the errors, I don't know why but you can check the tests results by clicking on `details` right next to them. I will try to take a look with it too tomorrow!

Reply:
Which is your local python version? On one of the failed checks, there is the following message:

```
Extension error:
Could not import extension configtraits (exception: cannot import name 'l_' from 'sphinx.locale' (/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/sphinx/locale/__init__.py))
Makefile:66: recipe for target 'html' failed
make: *** [html] Error 2
make: Leaving directory '/home/travis/build/ipython/ipython/docs'
The command "if [[ "$TRAVIS_PYTHON_VERSION" == "3.7" ]] && [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
  pip install -r docs/requirements.txt
  python tools/fixup_whats_new_pr.py
  make -C docs/ html SPHINXOPTS="-W"
fi
" exited with 2.
```

Reply:
Anaconda python3.7 is my local python version.
But why my local python version affects travis-CI checks?